Client/Frontend/Browser/Tabs/TabTrayViewController.swift

```diff
@@ -435,6 +436,7 @@ class TabTrayViewController: UIViewController,
 
     func remotePanel(didSelectURL url: URL, visitType: VisitType) {
         TelemetryWrapper.recordEvent(category: .action, method: .open, object: .syncTab)
+        // TODO: Need to provide handler for didSelectURL. Forthcoming.
```

Comment:
nit: TODO's should have a ticket number

Reply:
✅ Added ticket numbers; I'm not 100% sure which epic subtasks will cover some of this work but tried to pick the one that looked most appropriate.